stamina-core/src/main/scala/stamina/Persisters.scala

```diff
@@ -28,6 +30,17 @@ case class Persisters(persisters: List[Persister[_, _]]) {
   // format: ON
 
   def ++(other: Persisters): Persisters = Persisters(persisters ++ other.persisters)
+
+  private def requireNoOverlappingTags() = {
+    val overlappingTags = persisters.groupBy(_.tag).filter(_._2.length > 1).mapValues(_.map(_.key))
+
+    require(
+      overlappingTags.isEmpty,
+      s"Overlapping persisters: " + join(overlappingTags.map(tuple ⇒ "Tags " + join(tuple._2) + " all persist " + tuple._1.runtimeClass))
```

Comment:
The reference to "Tags" might be a bit confusing to most users. How about "Persisters with keys '..', '..' all persist''?